### ISSUE TITLE
test failure

### ISSUE THREAD

**Issue by wwood** [rubyworks/hashery#25]
Hi,

I'm trying to maintain hashery for the GNU Guix package manager, and I noticed on the upgrade to ruby 2.3.1 (or possibly 2.3.0) there was a test failure:
```
$ rubytest -Ilib -Itest test/
................................F.....................................................................................................................................................

Finished in 0.05111s, 3561.02 tests/s.

FAILURES

    
    (assertion) {}
    test/case_key_hash.rb:47
    45     test do
    46       s = KeyHash.new
 => 47       assert(s)
    48     end
    49   end
    test/case_key_hash.rb:47
    /gnu/store/jdpgw3951m4zlc4chkyy3sm6siqzp1iq-ruby-rubytest-cli-0.2.0/lib/ruby/gems/2.3.0/gems/rubytest-cli-0.2.0/lib/rubytest-cli.rb:48
    /gnu/store/jdpgw3951m4zlc4chkyy3sm6siqzp1iq-ruby-rubytest-cli-0.2.0/lib/ruby/gems/2.3.0/gems/rubytest-cli-0.2.0/lib/rubytest-cli.rb:18
```
I think this might stem from ae, through this testing.
```
irb(main):001:0> require 'ae'
=> true
irb(main):002:0> assert true
=> true
irb(main):003:0> assert {}  
AE::Assertion: (assertion) #<Proc:0x0056017f10b9a0@(irb):3>
	from (irb):3
	from /gnu/store/zg1fmaj0kl0c2hfqbyq7iahgrf2fb41a-profile/bin/irb:11:in `<main>'
```
I'm not sure what has changed in ruby exactly, do you have any ideas?

Thanks,
ben

**Comment by trans** [comment]
Thanks for reporting. I am guessing that the behavior of splatting has changed, so the hash object being passed to the `assert` method wasn't treating it as the simple "truthiness" parameter the test expected. It's not the proper way to test such a thing anyway, so I fixed it and released a new version 2.1.2.

:+1: For your work on GNU Guix.

**Comment by wwood** [comment]
Excellent. Thanks for being a good maintainer.
